### PR TITLE
Version bump, also don't make assumptions about future versions

### PR DESCRIPTION
**Pull request by kevinhowbrook**
Fixes https://github.com/torchbox/wagtail-markdown/issues/53

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def get_git_revision_hash():
 INSTALL_REQUIRES = [
     'Markdown>=2.6,<2.7',
     'bleach>=1.4.2,<2.2',
-    'Wagtail<2.3',
+    'Wagtail>=1.10,<2.5',
```
Comment (zerolab) [inline setup.py]: hmm... people reported this works with 2.5 too

Comment (kevinhowbrook) [reply setup.py]: what about `<=2.5`. or `<2.5.1` (current)
I'd like to remove this dependancy altogether.

Comment (zerolab) [reply setup.py]: why not `>=1.10`?

if a new Wagtail version breaks this, people will shout and then we can address it
wagtailmedia has `>2.0`

Comment (kevinhowbrook) [reply setup.py]: Yea probably the best way forward. Thanks @zerolab

Comment (zerolab) [reply setup.py]: thinking more of this, let's go `>=2.0`

Comment (kevinhowbrook) [reply setup.py]: Happy with that, just need to address travis

File: wagtailmarkdown/mdx/tables/__init__.py [hunk 2]
```diff
@@ -14,8 +14,8 @@
 Copyright 2009 - [Waylan Limberg](http://achinghead.com)
 """
 
-from __future__ import absolute_import
-from __future__ import unicode_literals
+from __future__ import absolute_import, unicode_literals
```
Comment (zerolab) [inline wagtailmarkdown/mdx/tables/__init__.py]: considering we are removing Python 2.7 support, we can get rid of `__future__` imports

https://python-future.org/unicode_literals.html

**Pull request merged**